abci/client/socket_client.go

```diff
@@ -412,6 +412,32 @@ func (cli *socketClient) FinalizeBlock(ctx context.Context, req *types.RequestFi
 	return reqRes.Response.GetFinalizeBlock(), cli.Error()
 }
 
+//
+// Side channel
```

Comment:
If we keep the comments, can we add the following?
`// TODO CHECK HEIMDALL-V2 check implementation of SideChannel`

Reply:
We won't keep sidechannel related code.